src/elfcore.c

```diff
@@ -1897,8 +1952,8 @@ int InternalGetCoreDump(void *frame, int num_threads, pid_t *pids,
   }
 
   /* scope */ {
-    int openmax  = sys_sysconf(_SC_OPEN_MAX);
-    int pagesize = sys_sysconf(_SC_PAGESIZE);
+    int openmax  = sysconf(_SC_OPEN_MAX);
+    int pagesize = sysconf(_SC_PAGESIZE);
```

Comment:
Why this change?

Reply:
The latest `linux_syscall_support.h` does not provide `sys_sysconf()`